public/js/components/Graph.js

```diff
@@ -11,29 +11,31 @@ export default class Graph extends React.Component {
     }
 
     componentDidMount() {
-        // TODO: Use actuial data
-        fetch(`keywords/trump`)
-            .then(res => res.json())
-            .then((dataPoints) => {
-                dataPoints = dataPoints
-                    .sort((a, b) => a.times - b.times)
+        fetch(`keywords/top`)
```

Comment:
@kirinrastogi Yeah it should be done with an action, I told @VikramBombhi in person but forgot to write it down. Good catch 👍